### ISSUE TITLE
Unable to override targets

### ISSUE THREAD

**Issue by pzuraq** [babel/ember-cli-babel#296]
There currently isn’t a way to override targets for a specific transpile tree, meaning it’s impossible to deviate from the main targets, though it _is_ possible to disable preset-env entirely. We should add a way to do this manually for use cases where it’s necessary.

**Comment by rwjblue** [comment]
Agree, I think tweaking our existing `ember-cli-babel` config options to allow `targets` should be good enough.
